src/compiler/factoryPublic.ts

```diff
@@ -2592,23 +2592,23 @@ namespace ts {
     }
 
     export function createJSDocAuthorTag(comment?: string) {
-        return createJSDocTag(SyntaxKind.JSDocAuthorTag, "author", comment);
+        return createJSDocTag<JSDocAuthorTag>(SyntaxKind.JSDocAuthorTag, "author", comment);

```

Comment:
I'm copying what's done elsewhere. This type argument is naughty, but your secret is safe with me (I won't tell Ryan).